dashboard/client/src/pages/job/TaskTimeline.tsx

```diff
@@ -0,0 +1,76 @@
+import {
+  Button,
+  createStyles,
+  makeStyles,
+  Typography,
+} from "@material-ui/core";
+import React from "react";
+import { RiDownload2Line } from "react-icons/ri";
+import { ClassNameProps } from "../../common/props";
+import { downloadTaskTimelineHref } from "../../service/task";
+
+const useStyle = makeStyles((theme) => ({
+  root: {
+    padding: theme.spacing(2, 0, 0),
+  },
+  button: {
+    marginTop: theme.spacing(2),
+  },
+}));
+
+type TaskTimelineProps = {
+  jobId: string;
+};
+
+export const TaskTimeline = ({ jobId }: TaskTimelineProps) => {
+  const classes = useStyle();
+
+  return (
+    <div className={classes.root}>
+      {/* TODO(aguo): Add link to external documentation about Timeline view. */}
+      <Typography>
+        Timeline view shows how tasks are executed across different nodes and
+        worker processes.
+        <br />
+        Download the trace file and analyze it by uploading it to{" "}
```

Comment:
> Download the trace file and analyze it by uploading it to **tools like** Perfetto UI

Could we add "tools like" here? It makes me like that we are not indicating that users have to use perfetto or chrome tracing.